Dockerfile

```diff
@@ -22,7 +24,7 @@ RUN curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/
 RUN trivy fs --format spdx-json --output /sbom.spdx.json /SBOM
 RUN trivy sbom /sbom.spdx.json --severity UNKNOWN,HIGH,CRITICAL --exit-code 1
 
-FROM node:20-alpine3.19
+FROM $BASE_IMAGE
```

Comment:
Same comment as https://github.com/hyperledger/firefly-tokens-erc20-erc721/pull/161/files#r1608741843

Reply:
fixing this with Build IMAGe ARG